python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_chaos_cleaning_strategy.py

```diff
@@ -502,14 +610,21 @@ def compute_cleaning_PCE(
 
 # %%
 #
-# These parameters lead to a total number of coefficients equal to 12. Among the 16 most significant coefficients, only 12 satisfy the criteria. Most of the coefficients have a small polynomial degree although some have a total degree equal as large as 7.
+# These parameters lead to a total number of coefficients equal to 12. Among   
+# the 16 most significant coefficients, only 12 satisfy the criteria. Most of   
+# the coefficients have a small polynomial degree although some have a total   
+# degree equal as large as 7.
 
 # %%
 #
 # Intermediate steps of the algorithm
 # -----------------------------------
 #
-# If we set the `verbose` optional input argument of the `compute_cleaning_PCE` function to `True`, then intermediate messages are printed in the Terminal (but not in the Jupyter output). For each step of the adaptivity algorithm, the code prints some of the internal parameters of the algorithm. The datastructure uses several variables that we now describe. 
+# If we set the `verbose` optional input argument of the   
+# `compute_cleaning_PCE` function to `True`, then intermediate messages are   
+# printed in the Terminal (but not in the Jupyter output). For each step of   
```

Comment:
Does it make sense to talk about the Jupyter output in a Python script? BTW, is that still true?